### DISCUSSION TITLE
Needed: kyverno-json test

### DISCUSSION THREAD

**Discussion by joebowbeer** [kyverno/kyverno-json#238] [General]
I'm looking at kyverno-json 0.0.1

I don't see a way to unit test the policies themselves. (Like there is in conftest and kyverno CLI.)

Should a `test` command be added?

Aside: What is the difference between kyverno-json and kyverno CLI? I suggest making this clearer in the overview docs.

**Comment by JimBugwadia** [comment]
The high level plan is to integrate kyverno-json with kyverno in release 1.12. Several details need to be discussed and finalized. All inputs and feedback is welcome!

**Comment by joebowbeer** [comment]
@JimBugwadia Should I create an issue to add a `test` command for unit testing policies? Or will this be handled by other tooling, e.g., kyverno for k8s?

Regarding integration with kyverno release 1.12, where is the discussion taking place? Just interested.
